lib/megaco/Makefile

```diff
@@ -115,7 +115,6 @@ include $(ERL_TOP)/make/otp_subdir.mk
 
 reconf:
 	(cd $(ERL_TOP) && \
-		./otp_build autoconf && \
 		./otp_build configure && \
 		cd $(ERL_TOP)/../libraries/megaco)
```

Comment:
@bmk this last `cd` (which in it self is a bit strange since it is the last command in a subshell) seems to be a leftover since the clearcase days when megaco was in a different VOB than erts.